<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0" columnminimumwidth="0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>7</number>
      </property>
      <item row="2" column="3" colspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QPlainTextEdit" name="textEdit_output"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;    明文&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;密文输入&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;密文输出&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPlainTextEdit" name="inputText"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_encrypt">
        <property name="text">
         <string>加密</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QPushButton" name="pushButton_decrypt">
        <property name="text">
         <string>解密</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;解密&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPlainTextEdit" name="textEdit_input"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_getpk">
        <property name="text">
         <string>查看私钥</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QPushButton" name="pushButton_getmod">
        <property name="text">
         <string>查看模数</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>textEdit_input</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>textEdit_output</tabstop>
  <tabstop>pushButton_decrypt</tabstop>
  <tabstop>pushButton_encrypt</tabstop>
  <tabstop>inputText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
